core/src/impl/Kokkos_Core.cpp

```diff
@@ -81,6 +81,19 @@ std::stack<hook_function_type, std::list<hook_function_type>> finalize_hooks;
 
 namespace Kokkos {
 namespace Impl {
+using metadata_category_type = std::string;
+using metadata_key_type      = std::string;
+using metadata_value_type    = std::string;
+
```

Comment:
Comment that category is only used to group related key-value pairs when printing the configuration and that the information is not passed to the tools.

Reply:
7230f06